articles/active-directory/develop/quickstart-v2-javascript-auth-code-react.md

```diff
@@ -0,0 +1,188 @@
+> 1. Sign in to the [Azure portal](https://portal.azure.com).
+> 1. If your account gives you access to more than one tenant, select the account at the top right, and then set your portal session to the Azure Active Directory (Azure AD) tenant you want to use.
+> 1. Select [App registrations](https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade/quickStartType/JavascriptSpaQuickstartPage/sourceType/docs).
+> 3. Enter a name for your application.
+> 1. Under **Supported account types**, select **Accounts in any organizational directory and personal Microsoft accounts**.
+> 1. Select **Register**.
+> 1. Go to the quickstart pane and follow the instructions to download and automatically configure your new application.-->
+
+>
+> ### Option 1 (Manual): Register and manually configure your application and code sample
+>
+> #### Step 1: Register your application
+>
+> 1. Sign in to the [Azure portal](https://portal.azure.com).
+> 1. If your account gives you access to more than one tenant, select your account at the top right, and then set your portal session to the Azure AD tenant you want to use.
+> 1. Select [App registrations](https://go.microsoft.com/fwlink/?linkid=2083908).
+> 1. Select **New registration**.
+> 1. When the **Register an application** page appears, enter a name for your application.
+> 1. Under **Supported account types**, select **Accounts in any organizational directory and personal Microsoft accounts**.
+> 1. Under **Redirect URI** select `Single-page application (SPA)`.
+> 1. Set the **Redirect URI** value to `http://localhost:3000/`. This is the default port NodeJS will listen on your local machine. We’ll return the authentication response to this URI after successfully authenticating the user 
+> 1. Select **Register**. On the app **Overview** page, note the **Application (client) ID** value for later use.
+> 1. In the left pane of the registered application, select **Authentication**.
+> 1. Under **Platform Configurations** expand **Single-page application**.
+> 1. Confirm that under **Grant types** ![Already configured](media/quickstart-v2-javascript/green-check.png)Your Redirect URI is eligible for the Authorization Code Flow with PKCE.
+
+#### Step 2: Download the project
+
+> [!div renderon="docs"]
+> To run the project with a web server by using Node.js, [download the core project files](https://github.com/Azure-Samples/ms-identity-javascript-react-spa/archive/main.zip).
+
+> [!div renderon="portal" class="sxs-lookup"]
+> Run the project with a web server by using Node.js
+
+> [!div renderon="portal" class="sxs-lookup" id="autoupdate" class="nextstepaction"]
+> [Download the code sample](https://github.com/Azure-Samples/ms-identity-javascript-react-spa/archive/main.zip)
+
+> [!div renderon="docs"]
+> #### Step 3: Configure your JavaScript app
+>
+> In the *app* folder, open the *authConfig.js* file and update the `clientID`, `authority`, and `redirectUri` values in the `msalConfig` object.
```

Comment:
I don't see an app folder.

Reply:
Is it [src/authConfig.js](https://github.com/Azure-Samples/ms-identity-javascript-react-spa/blob/main/src/authConfig.js)